### ISSUE TITLE
Alt tags for thumbnail galleries  

### ISSUE THREAD

**Issue by vkozintsev** [netrivet/prophoto-issues#750]
There is no alt tags for images when we create a gallery as the thumbnail grid. I would love to see the Media alt text used here.
![alt_tag](https://cloud.githubusercontent.com/assets/26632225/24261841/ae46248e-0fce-11e7-8f30-936754cbbef8.png)

**Comment by coreypotter** [comment]
@vkozintsev can you add this over on Ticket 737? https://github.com/netrivet/prophoto-issues/issues/737

**Comment by brianium** [comment]
I've moved @vkozintsev's comment to #737 - closing this as a duplicate
